strawberry/schema/types/base_scalars.py

```diff
@@ -23,7 +23,7 @@ def inner(value: str):
 
 def parse_decimal(value: str) -> decimal.Decimal:
```

Comment:
this says `:str` but actually could be anything. Should we change to `any` or something more specific? this could be a string, a float, an int 🤔

Reply:
I think `: object` is best

Reply:
👌  changed